src/config/dynamic-theme-fixes.config

```
@@ -747,6 +747,14 @@ IGNORE IMAGE ANALYSIS
 
 ================================
 
+apclassroom.collegeboard.org
+
+INVERT
+a.overview-icon svg
+button.Zoom--expander svg g > *:first-child
```

```suggestion
button.Zoom--expander svg g > :first-child
```
In the world of CSS no need to specify the `*` in this case.